src/frontend/planner_test/tests/testdata/expr.yaml

```diff
@@ -144,20 +144,19 @@
     -- Single quoted literal can be treated as number without error.
     values(round('123'));
   batch_plan: |
-    BatchValues { rows: [[Round(123:Float64)]] }
+    BatchValues { rows: [[123:Float64]] }
```

Comment:
maybe let's move const_eval into separate PR so it's easier to review

Reply:
This PR will be for const eval, the exprrewritable part is now here: https://github.com/risingwavelabs/risingwave/pull/7542